solvebio/query.py

```diff
@@ -1030,6 +1064,8 @@ def __init__(
             result_class=dict,
             debug=False,
             error=None,
```

Comment:
can you remove this? I don't think it is used actually (for input purposes)

Also, reorder the docstring so it matches the order of the params.

Reply:
Done.